primitives/src/sentry.rs

```diff
@@ -80,10 +80,16 @@ pub struct SuccessResponse {
 }
 
 #[derive(Serialize, Deserialize, Debug)]
-pub struct ValidatorMessageResponse {
+pub struct ValidatorMessage {
     pub from: String,
     pub received: DateTime<Utc>,
-    pub msg: Vec<MessageTypes>,
+    pub msg: MessageTypes,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub enum ValidatorMessageResponse {
```

Comment:
I think this should be in sentry_interface and not public as it's only used for deserializing in `sentry_interface`

Reply:
it's needed it in sentry too for `ValidatorMessageResponse`